frontend/src/components/c-summary-charts.vue

```diff
@@ -372,7 +372,13 @@ export default defineComponent({
       default: undefined,
     },
   },
-  data() {
+  data(): {
+    drags: Array<number>,
+      activeRepo: string | null,
+      activeUser: string | null,
+      activeTabType: string | null,
+      isTabOnMergedGroup: boolean,
+      } {
```

Comment:
Can you fix the space formatting here?

Reply:
Gradle's linter prevents the build if the space formatting is changed so I'll keep it unchanged for now